var/spack/repos/builtin/packages/libxkbcommon/package.py

```diff
@@ -39,6 +41,9 @@ class Libxkbcommon(MesonPackage, AutotoolsPackage):
 
     variant("wayland", default=False, description="Enable Wayland support")
 
+    depends_on("meson@0.41:", type="build", when="@0.8:")
```

Comment:
Since 0.8.2 is using build system autotools, we shouldn't require meson.
```suggestion
    depends_on("meson@0.41:", type="build", when="@0.9:")
```